src/store/configureStore 3.js

```diff
@@ -0,0 +1,12 @@
+import {createStore, compose, applyMiddleware} from 'redux';
+import rootReducer from '../reducers/rootReducer';
+import thunk from 'redux-thunk';
+
+const composeEnhancer = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+
+export default function configureStore() {
+    return createStore(
+        rootReducer,
+        composeEnhancer(applyMiddleware(thunk))
+    );
+}
```

Comment:
Why do you have this? We have an identical file in this folder already. This should be deleted.